src/client/FlagInput.ts

```
@@ -76,65 +53,44 @@ export class FlagInput extends LitElement {
-          : ""}
       </div>
     `;
   }
+
+  updated() {
+    const preview = this.renderRoot.querySelector(
+      "#flag-preview",
+    ) as HTMLElement;
+    if (preview) {
+      preview.innerHTML = "";
+      if (this.flag && this.flag.startsWith("!")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)